.pre-commit-hooks.yaml

```
@@ -44,6 +44,7 @@
   name: flake8-lint
   description: '`flake8` is a command-line utility for enforcing style consistency across Python projects.'
   entry: 'flake8 --ignore=E501,W503'
+  additional_dependencies: [flake8-print]
   language: system
```

This isn't supported for "system". 

https://pre-commit.com/#system
```suggestion
  language: system
```